components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/definitions/OAS2Parser.java

```diff
@@ -574,10 +574,13 @@ private void updateOperationManagedInfo(SwaggerData.Resource resource, Operation
         }
         operation.setVendorExtension(APIConstants.SWAGGER_X_AUTH_TYPE, authType);
         operation.setVendorExtension(APIConstants.SWAGGER_X_THROTTLING_TIER, resource.getPolicy());
-        // AWS Lambda: set arn to swagger
+        // AWS Lambda: set arn & timeout to swagger
         if (resource.getAmznResourceName() != null) {
             operation.setVendorExtension(APIConstants.SWAGGER_X_AMZN_RESOURCE_NAME, resource.getAmznResourceName());
         }
+        if (resource.getAmznResourceTimeout() != 0) {
```

Comment:
What happen if resource.getAmznResourceTimeout() null?

Reply:
if not initialized it'll be 0